### PR TITLE
fix(@community): update create community modal

### LINKED ISSUE #5840: Update create community modal
### Description

figma: https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=2563%3A543536

- [x] margins need to fit the figma spec
- [x] Thumbnail image should say community logo
- [x] Change `Create` to `Next` button text when we add the corresponding community screens flow defined in figma

notes

- The community popup should use the new [CommunityEditSettingsPanel](https://github.com/status-im/status-desktop/blob/master/ui/app/AppLayouts/Chat/panels/communities/CommunityEditSettingsPanel.qml)
  - Now, we duplicate the same functionality also in the old [CreateCommunityPopup](https://github.com/status-im/status-desktop/blob/master/ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml)
  - [CommunityEditSettingsPanel](https://github.com/status-im/status-desktop/blob/master/ui/app/AppLayouts/Chat/panels/communities/CommunityEditSettingsPanel.qml) has Banner as extra
  - [CreateCommunityPopup](https://github.com/status-im/status-desktop/blob/master/ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml) has two more checkboxes as extra
- community banner is out of scope, is already being done in another task

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #5840

### What does the PR do

Update UI of CreateCommunityPopup

### Affected areas

Communities

### Screenshot of functionality

image

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml
Comment (MishkaRogachev) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:136]: Here we need to get info about `pinMessagesAllMembers` and others flags. I don't think rootStore is good place for that. We can:
- Pass and edit whole Community object, parsed to json or just QObject-based model.
- Make editCommunity behave with empty args in way "leave it as it"

Comment (stefandunca) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:136]: That is a good idea! It seems out of scope for this PR, though. The best way is to create a follow-up task and tackle it there leaving this PR as a simple refactoring.

**Review by stefandunca — APPROVED** [review]
Great job, looking forward to seeing it in action after it is finished

**Comment by stefandunca** [comment]
@MishkaRogachev beware that this change conflicts with the following two changes: #5437 #5895

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:136]: Thanks, think in the same way.  Can i just leave here true and this FIXME? Informing @iurimatias

**Comment by MishkaRogachev** [comment]
> @MishkaRogachev note that this change conflicts with the following two changes: #5437 #5895

Yes, so probably who will be the last should resolve it. Btw I can hold this PR until baner & logo merged.

**Comment by stefandunca** [comment]
> Yes, so probably who will be the last should resolve it. Btw I can hold this PR until baner & logo merged.

I kind of got used to fixing conflicts and migrating functionality :). It depends on too many moving pieces, so don't wait for it :)

**Review by endulab — APPROVED** [review]

**Comment by osmaczko** [comment]
This is completely conflicting with my PR :sob: Please see: https://github.com/status-im/status-desktop/pull/5882

https://user-images.githubusercontent.com/33099791/170685746-4c164038-72aa-45f1-b865-63ff8481c24d.mp4

**Comment by MishkaRogachev** [comment]
> This is completely conflicting with my PR :sob: Please see: https://github.com/status-im/status-desktop/pull/5882
> 
> https://user-images.githubusercontent.com/33099791/170685746-4c164038-72aa-45f1-b865-63ff8481c24d.mp4

I can wait for yours and @stefandunca PRs merged and when rewrite mine

**Comment by osmaczko** [comment]
> I can wait for yours and @stefandunca PRs merged and when rewrite mine

Thank you!

**Comment by John-44** [comment]
> @MishkaRogachev please adapt the names of the fields to the figma ones:
> 
> image
> 
> also, @John-44 should this modal be autoscalable? it looks odd on my small 13 inch screen -> kinda glued to the top without margins
> 
> image

Yes, this modal should go to the full height of the window (but with a 64px margin top and bottom) if the window isn't tall enough to display the modal content without scrolling (which will almost always be the case with this modal, unless a user is using a monitor in portrait mode or something like that).

This same behaviour should apply to most tall modals in status desktop

**Pull request closed without merging**
